deepomatic/cli/cmds/site/intervention/create.py

```diff
@@ -0,0 +1,23 @@
+from ...utils import Command, BuildDict
+from ..utils import SiteManager
+
+
+class CreateCommand(Command):
+    """
+        Create a new intervention
+    """
+
+    def setup(self, subparsers):
+        parser = super(CreateCommand, self).setup(subparsers)
+        parser.add_argument('-u', "--base_url", required=True, type=str, help="url of your api")
+        parser.add_argument('-n', '--name', required=True, type=str, help="Intervention name")
+        parser.add_argument('-m', "--metadata", dest='metadata',
+                            action=BuildDict, nargs="+", metavar="NAME:VAL",
+                            help='Metadata for the intervention in format name:value')
+        return parser
+
+    def run(self, base_url, name, metadata, **kwargs):
+        metadata = {'version': '1.1',
+                    # NOTE: Temporary field to pass the question to the CustomerAPI
+                    'question_ids': ['1c27cba0-5fb7-4417-9fd1-683bc70df8ce']}
```

Comment:
```suggestion
        metadata = {'version': '1.1'}
```
Question id not necessary anymore

Reply:
yes ! a relic of the past :D